packages/gatsby/src/commands/develop-process.ts

```diff
@@ -210,57 +230,104 @@ module.exports = async (program: IProgram): Promise<void> => {
       },
       doingEverythingElse: {
         invoke: {
-          src: async ({ workerPool, store, app }): Promise<void> => {
+          src: async (): Promise<void> => {
             // All the stuff that's not in the state machine yet
 
-            await writeOutRequires({ store })
             boundActionCreators.setProgramStatus(
               ProgramStatus.BOOTSTRAP_QUERY_RUNNING_FINISHED
             )
 
             await db.saveState()
 
-            await waitUntilAllJobsComplete()
-            requiresWriter.startListener()
             db.startAutosave()
```

Comment:
Do we still need autosave? Can't we add it at specific times in the state machine? Right before we go to idle/waiting?